spec/models/quiz_spec.rb

```diff
@@ -5,4 +5,29 @@
 
   it { should validate_presence_of(:name) }
   it { should validate_uniqueness_of(:name) }
+
+  it "returns true if it has been completed by a student" do
+    user = FactoryGirl.create(:user_with_answered_quiz)
+    quiz = Quiz.first
```

Comment:
We can't guarantee that the first quiz is the one we want. Use the same strategy as in your user model for creating a quiz and answer submissions.